src/controllers/PacienteController.ts

```diff
@@ -18,6 +18,17 @@ class PacienteController {
 
         const pacientesRepository = getCustomRepository(PacientesRepository);
 
+        const pacienteAlreadyExists = await pacientesRepository.findOne({
+            cpf,
+            nsus
+        })
```

Comment:
Essa verificação funciona como "and" ou como "or"?

Reply:
Tipo, se só o cpf passado já for de um outro paciente, será que encontra?

Reply:
ou o findOne só vai retornar o paciente que tenha tanto o cpf quanto o nsus passado?

Reply:
funciona como and, porém vi agora que a formatação tá errada pra funcionar aceitando os dois

Reply:
Você quer que seja como or? Se for, eu altero aqui

Reply:
Acredito que o idela seja como um or, porque dessa forma evitamos que cpf e nsus sejam duplicado

Reply:
Quando terminar, lembra de solicitar revisão novamente ;)

Reply:
Logo, o ideal é que na mesma consulta ambos os casos sejam tratados. [Nesse tópico  aqui](https://github.com/typeorm/typeorm/issues/2929) eu encontrei uma possível solução: 

`let results = await userRepo.find({`
`    where : [`
`        {firstname : Like('%John%')},`
`        {lastname : Like('%Doe%')}`
`    ]`
`});`

Se você puder dar uma olhada se realmente funciona pro nosso caso, agradeço.

(lembra de adicionar a indentação)